build/BUILD.ada-url

```diff
@@ -0,0 +1,13 @@
+
+cc_library(
+    name = "ada-url",
+    hdrs = ["ada.h", "ada_c.h"],
+    srcs = ["ada.cpp"],
+    visibility = ["//visibility:public"],
+    include_prefix = ".",
+    copts = ["-w", "-msse2"],
+    defines = [
+      "ADA_SSE2=1"
+    ],
+    alwayslink = 1,
```

Comment:
alwayslink should not be needed here (unless the library uses some linking/visibility dark magic), it makes dead code elimination more difficult.

Reply:
Yeah, another experiment while I was seeing what was needed to build :-)